ci.not.Rune.yml

```
@@ -0,0 +1,61 @@
+app-id: ci.not.Rune
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: rune
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --own-name=org.mpris.MediaPlayer2.rune_player.*
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: rune
+    buildsystem: simple
+    build-commands:
+      - sed -Ei 's#(Icon=).+#\1ci.not.Rune#' assets/rune.desktop
+      - mkdir -p /app/share/applications
+      - cp assets/rune.desktop /app/share/applications/ci.not.Rune.desktop
+      - sed -Ei 's#(<id>)[a-z]+#\1ci.not.Rune#' assets/rune.metainfo.xml
+      - sed -Ei 's#(<launchable type="desktop-id">)[a-z\.]+#\1ci.not.Rune.desktop#'
+        assets/rune.metainfo.xml
+      - mkdir -p /app/share/metainfo
+      - cp assets/rune.metainfo.xml /app/share/metainfo/ci.not.Rune.metainfo.xml
```

```suggestion
      - install -Dm644 assets/rune.desktop -t /app/share/applications/
      - install -Dm644 assets/rune.metainfo.xml -t /app/share/metainfo/
```

And then add in the top of the manifest
`rename-desktop-file: rune.desktop
`rename-appdata-file: rune.metainfo.xml`

---

`rename-appdata-file` doesn't seem to be able to change the `<launchable type="desktop-id"></launchable>` field, can I keep the sed command for that line?

---

that's done my `rename-desktop-file`. because it has to match with the desktop-file